packages/docmanager-extension/src/index.ts

```diff
@@ -257,6 +260,18 @@ function addCommands(app: JupyterLab, docManager: IDocumentManager, palette: ICo
     mnemonic: args => args['mnemonic'] as number || -1
   });
 
+  commands.addCommand(CommandIDs.openTab, {
+    execute: args => {
+      const path = typeof args['path'] === 'undefined' ? ''
+        : args['path'] as string;
+      return docManager.services.contents.getDownloadUrl(path).then(url => {
+        window.open(url, '_blank');
```

Comment:
Out of curiosity, does this also work for PDFs?

Reply:
Yes.

![new-tab-pdf](https://user-images.githubusercontent.com/591645/38510086-3208a940-3c24-11e8-923f-5483f2641baa.gif)